packages/metatransaction-broadcaster/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "metatransaction-broadcaster",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Redundant I know but would be nice to fill this in.

Reply:
With what, exactly?

Reply:
Nothing, there's no other test commands.